app/models/edition/role_appointments.rb

```diff
@@ -21,4 +21,11 @@ def can_be_associated_with_role_appointments?
   def search_index
     super.merge("people" => role_appointments.map(&:slug))
   end
+
+  def is_associated_with_a_minister?
```

Comment:
Sure, it was just something that occurred to me that "might" make sense for safety. At present, edition classes only ever mix in one of `Edition::Ministers`, `Edition;:Appointment`, or `Edition::RoleAppointments`, so we're fine.

Thinking on it, it does seem unlikely that we'll change a document type so that it has more than one of these relationships, but if we ever did, we'll have a bug. That said, the interaction between the associations and join models would get pretty tangled at that point so it'd probably be the least of our worries, so maybe it's unnecessary right now.